graphene_django/fields.py

```diff
@@ -39,10 +39,8 @@ def list_resolver(django_object_type, resolver, root, info, **args):
         if queryset is None:
             # Default to Django Model queryset
             # N.B. This happens if DjangoListField is used in the top level Query object
-            model = django_object_type._meta.model
-            queryset = maybe_queryset(
-                django_object_type.get_queryset(model.objects, info)
-            )
+            model = django_object_type._meta.model.objects
```

Comment:
```suggestion
            model_manager = django_object_type._meta.model.objects
```

Bit of a nit pick but the variable name `model` is misleading here.